main_test.go

```diff
@@ -184,7 +184,7 @@ func TestPublishersEndpoints(t *testing.T) {
 		},
 		{
 			query: "POST /v1/publishers",
-			body:  `{"URL":"https://www.example.com", "email":"example@example.com"}`,
+			body:  `{"codeHosting": [{"url" : "https://www.example.com"}], "email":"example@example.com"}`,
```

Comment:
Pagination will come in another PR, right?

Reply:
yep!